### PR TITLE
[Mission-Control] Added missing steps in upgrade notes for 1x to 2x chart versions

### PR DESCRIPTION
**Pull request by rahulsadanandan**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/mission-control/UPGRADE_NOTES.md [hunk 1]
```diff
@@ -44,16 +44,18 @@ This file describes special upgrade notes needed at specific versions
...
-          b. Once logged in, create a dump file from the previous database using pg_dump, connect to previous postgresql chart:\
+          b. Once logged in, create the dump files from the previous database using pg_dump, connect to previous postgresql chart:\
            ```bash
-           $ pg_dump -h <OLD_PG_SERVICE_NAME> -U mission-control DATABASE_NAME > /tmp/backup.sql
+           $ pg_dumpall --roles-only -U postgres -h <OLD_PG_SERVICE_NAME> --clean --file=/tmp/roles.sql
+           $ pg_dump -h <OLD_PG_SERVICE_NAME> -U postgres DATABASE_NAME > /tmp/backup.sql
            ```
-          c. After you ran above command you should be prompted for a password, this password is the previous chart password (OLD_PG_PASSWORD). This operation could take some time depending on the database size.\
-          d. Once you have the backup file, you can restore it with a command like the one below:\
+          c. When you run the above commands you should be prompted for a password, this password is the previous chart password (OLD_PG_PASSWORD). This operations could take some time depending on the database size.\
```
Comment (danielezer) [inline stable/mission-control/UPGRADE_NOTES.md:52]: These operations instead of this

Comment (rahulsadanandan) [reply stable/mission-control/UPGRADE_NOTES.md]: Corrected now.

File: stable/mission-control/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mission-control
 description: A Helm chart for JFrog Mission Control
-version: 2.0.1
+version: 2.0.2
```
Comment (chukka) [inline stable/mission-control/Chart.yaml:4]: @rahulsadanandan can you please add this chart version in chart install step in readme file

Comment (rahulsadanandan) [reply stable/mission-control/Chart.yaml:4]: Its done now

**Pull request merged**